src/coreComponents/common/format/table/TableFormatter.cpp

```diff
@@ -67,7 +68,8 @@ string TableCSVFormatter::dataToString( TableData const & tableData ) const
   return oss.str();
 }
 
-string TableCSVFormatter::toString( TableData const & tableData ) const
+template<>
+string TableCSVFormatter::toString< TableData >( TableData const & tableData ) const
```

Comment:
what is the benefit of the template specialization here vs overload?

Reply:
Formatting with a `TableFunction`  is very different from formatting with a normal data source and is heavily coupled to `TableFunctions,` I did that to avoid to add dependancies to `TableFunction`